README.md

```diff
@@ -6,6 +6,8 @@
 
 > Don't forget to [read our contributing guide](https://github.com/OtacilioN/awesome-hacktoberfest-2020/blob/master/CONTRIBUTING.md)
 
+> Please read the [updated guidlines](https://hacktoberfest.digitalocean.com/hacktoberfest-update) of Hacktoberfest 2020 if you don't to want to get spammed or marked invalid.
```

Comment:
I'm not sure about this disclaimer

Reply:
It is from the official site of the digital ocean and is completely legit.
Due to hacktoberfest people are just making PR's without thinking.
So they came up with updated guidelines. I just helped you so that people can read it and your work becomes easier.